declarative-examples/simple-examples/credentialsMixedEnvironmentand.groovy

```diff
@@ -0,0 +1,28 @@
+pipeline {
```

Comment:
removed "and" at end.  I pulled "credentials" from end and moved it to the beginning to make the naming work with the other credentials example.